### PR TITLE
Fixes #73: Adapter for Firebase Installations SDK 

### LINKED ISSUE #73: Adapter for `firebaseinstallations.googleapis.com`
Endpoint URL: `https://firebaseinstallations.googleapis.com/v1/projects/*/installations`

Sample requests:

* [worrying-confessions,40755](http://localhost:8001/data/requests/worrying-confessions,40755)
* [informed-consent,563944](http://localhost:8001/data/requests/informed-consent,563944)
* [do-they-track,81657](http://localhost:8001/data/requests/do-they-track,81657)
* [ios-watching-you,33759](http://localhost:8001/data/requests/ios-watching-you,33759)
* [monkey-april-2024,734](http://localhost:8001/data/requests/monkey-april-2024,734)
* [monkey-july-2023,75503](http://localhost:8001/data/requests/monkey-july-2023,75503)

### PR DESCRIPTION
**Pull request by baltpeter**
This is based on #59, which needs to be merged first.

### CODE REVIEW

File: src/adapters/google.ts [hunk 1]
```diff
@@ -1315,4 +1315,52 @@ export const adapters: Adapter[] = [
...
     },
+
+    {
+        slug: 'firebaseinstallations',
+        name: 'Firebase Installations SDK',
+        description: 'firebaseinstallations',
+        tracker,
+
+        endpointUrls: [/^https:\/\/firebaseinstallations\.googleapis\.com\/v1\/projects\/.+\/installations\/?$/],
+        match: (r) => r.content?.startsWith('{"'),
```
Comment (zner0L) [inline src/adapters/google.ts:1326]: Why do we need this `match` clause? There is no other adapter for the same endpoint.

**Review by zner0L — CHANGES_REQUESTED** [review]

File: src/adapters/google.ts (hunk 1 above)
Comment (baltpeter) [reply src/adapters/google.ts:1326]: https://github.com/tweaselORG/TrackHAR/issues/73#issuecomment-2093091985

Comment (zner0L) [reply src/adapters/google.ts:1326]: Hm, I see. So we would need to parse the Js as well in some way.

**Review by zner0L — APPROVED** [review]

File: src/adapters/google.ts (hunk 1 above)
Comment (baltpeter) [reply src/adapters/google.ts:1326]: Not for a single request. :D

**Pull request merged**